routes/chat.js

```diff
@@ -32,7 +33,7 @@ const getChannelDetail = require('../controllers/chat/getChannelDetail');
 router.get('/create-channel', chatController.createChannel);
 router.post('/add-moderator', chatController.addChannelModerator);
 router.post('/add-members-channel', auth.isAuth, chatController.addMembers);
-router.post('/anonymous', auth.isAuthAnonim, chatController.sendAnonymous);
+router.post('/anonymous', auth.isAuthAnonim, sendAnonymousMessage);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The route handler for `/anonymous` has been changed from `chatController.sendAnonymous` to `sendAnonymousMessage`. Make sure that `sendAnonymousMessage` has the same signature as the old `chatController.sendAnonymous` to avoid breaking changes.
```diff
- router.post('/anonymous', auth.isAuthAnonim, chatController.sendAnonymous);
+ router.post('/anonymous', auth.isAuthAnonim, sendAnonymousMessage);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->